src/Assimp/Silk.NET.Assimp/Structs/AABB.gen.cs

```diff
@@ -18,9 +18,9 @@
 namespace Silk.NET.Assimp
 {
     [NativeName("Name", "aiAABB")]
-    public unsafe partial struct AABB
+    public unsafe partial struct Aabb
```

Comment:
idk what you did to name rules, but that's certainly pretty questionable

Reply:
This makes the bindings consistent with the rest of Silk.NET (i.e. `Sdl`, `Glfw`, etc). The naming changes were required to give some DirectX types nice names. Moot anyway, will be replaced with Box3 when maths is in.

Reply:
Ok. if it isn't too much of a difficulty make the file name consistent please.

Reply:
It's pretty difficult given Windows ignores filenames, would require manual git index changes.